docs/bash-scripting-cheatsheet.md

```diff
@@ -148,7 +148,7 @@ Using a wildcard that you don't want globbed because you're passing it to
 "*.json foo-${VAR}-*.json"
 ```
 
-### For loops are bad while while loops are good
+### For loops are bad - while loops are good
```

Comment:
This was a fairly manual change, my default was to just drop a `while`

Reply:
This was an attempt at cleverness; it might have been better structured by me as `"for loops" are bad while "while loops" are good`.  That said, I already made myself laugh and making this easier for humans to parse and automation to do its job seems preferable, so I think this change makes sense / is appropriate, thank you!